src/calculateRentalCost.js

```diff
@@ -29,6 +29,21 @@
 
 function calculateRentalCost(days) {
   // write code here
+  const pricePerDay = 40;
+  const smallDiscount = 20;
+  const basicDiscount = 50;
+  const smallDiscountDays = 3;
+  const basicDiscountDays = 7;
+
+  if (days >= basicDiscountDays) {
+    return days * pricePerDay - basicDiscount;
+  }
+
+  if (days >= smallDiscountDays && days < basicDiscountDays) {
+    return days * pricePerDay - smallDiscount;
+  }
```

Comment:
Since the days >= basicDiscountDays and days >= smallDiscountDays && days < basicDiscountDays conditions are mutually exclusive, you can replace the second if statement with an else if statement. This makes it clear that only one of these conditions will be met.

But it's still good this way :)